modules/badge/react/index.ts

```diff
@@ -0,0 +1 @@
+export {default as CountBadge, CountBadgeProps} from './lib/CountBadge';
```

Comment:
```suggestion
import CountBadge from './lib/CountBadge';

export default CountBadge;
export {CountBadge};
export * from './lib/CountBadge';
```

This should be aligned with the other modules right?

Reply:
I assume we want to export CountBadge and CountBadgeProps, but not as default. I wouldn't want someone to `import CountBadge from 'ckr-badge'` if we later create a generic Badge that we want to export as default.

Reply:
I think we should consider deprecating default imports/exports

Reply:
> I think we should consider deprecating default imports/exports

I would love to! Maybe for v5?